ieeg_visual/sub-01/ses-01/ieeg/sub-01_ses-01_task-visual_run-01_channels.tsv

```diff
@@ -1,119 +1,119 @@
 name	type	units	low_cutoff	high_cutoff	reference	group	status
-1	ECoG	µV	300	0.5	intracranial	1	good
```

Comment:
@dorahermes does this dataset need to be updated somewhere? i.e., the "real" dataset? --> just to keep the real data and the examples in sync

Reply:
We've only shared the non-BIDS version currently, we plan to put the BIDS data on openneuro. Will make sure to  include these updates!